build.sbt

```diff
@@ -159,7 +159,16 @@ lazy val lib = (project in file("lib"))
     defaultSettingsWithPlugin,
     name := "SpinalHDL-lib",
     libraryDependencies += "commons-io" % "commons-io" % "2.11.0",
+    libraryDependencies ++= Seq(
+      "com.fasterxml.jackson.core" % "jackson-core" % "2.16.0-rc1",
+      "com.fasterxml.jackson.core" % "jackson-annotations" % "2.16.0-rc1",
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.16.0-rc1",
+      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.16.0-rc1",
```

Comment:
Does the rc version is necessary, I suggest to use a stable version. How about 2.15.3?

Reply:
I switch to latest stable version, it works well.